tests/integration/helpers/cluster.py

```diff
@@ -1369,7 +1389,11 @@ def create_dir(self, destroy_dir=True):
 
         print("Copy common configuration from helpers")
         # The file is named with 0_ prefix to be processed before other configuration overloads.
-        shutil.copy(p.join(HELPERS_DIR, '0_common_instance_config.xml'), self.config_d_dir)
+        if self.copy_users_xml:
+            shutil.copy(p.join(HELPERS_DIR, '0_common_instance_config.xml'), self.config_d_dir)
+        else:
+            shutil.copy(p.join(HELPERS_DIR, '0_common_instance_config.yaml'), self.config_d_dir)
```

Comment:
It seems we need a third variable here like copy_common_configs, which will copy `0_common_*.xml` only if it's True.

Reply:
Actually, the 0_common_* file variant we want to copy depends on users.* extension. If we have users.xml we need to copy 0_common_*.xml. In case we have users.yaml, we'll need to copy 0_common_*.yaml. Is it really necessary to create another argument here?

Reply:
No, I meant if `copy_common_configs` is True then we copy `*.xml`. And when it's false we copy nothing here. `0_common_instance_config.yaml` can be copied in your test if you append `0_common_instance_config.yaml` to the list specified in the parameter `main_configs` which you pass to the function `add_instance()`.